internal/tiltfile/tiltfile.go

```diff
@@ -243,6 +246,12 @@ func (tfl *tiltfileLoader) reportTiltfileLoaded(
 			tags[fmt.Sprintf("tiltfile.invoked.%s.arg.%s", builtinName, argName)] = strconv.Itoa(count)
 		}
 	}
+	for k, v := range features {
+		if v.Status == feature.Active && v.Enabled {
```

Comment:
feels like we should only report this when it's different than the default (if the intention is to only report when people have opted in)

Reply:
I briefly considered that, but wasn't able to come up with any use cases for it. My thinking is that for questions like "of users who have this feature enabled, how many have interacted with it?", this saves us the step of checking the version number, at relatively close to launch. Once we change the flag to obsolete, we have to check the version number anyway, so maybe it doesn't buy us that much.

Reply:
the pr description says "to track how many users have opted into experimental features"...so are you saying that's outdated?

if you're trying to answer the question "of users who have this feature enabled, how many have interacted with it?", it seems like this would make that harder? you'd have to do some aggregations and joins of ​the "interacted" event against the "tiltfile.loaded" event, right?

a pattern i've seen is to create a manually-curated fact table for which feature launched with which version, then use that? seems like it'd be simpler, since we already ship a version on every event. unless i'm missing something?

Reply:
i could see a version table with feature list being useful in the future!

fwiw, i think including a list of enabled features on tilt up (regardless of explicit user opt-in) is fine and useful. it would allow us to track how many people have access to a feature, whether they explicitly opted into it or it's turned on by default. 

this current approach may make it more complicated if we did want to analyze how many users opted into a specific feature before it was turned on by default.

though, if we only counted explicit opt-ins, i could see instances where a user opts into a feature, then upgrades tilt to a version where that's turned on by default, but they don't remove that call from their `Tiltfile`, so analyzing explicit opt-ins would still require version comparison. 

(hopefully this isn't too much bike-shedding. i'm outlining specific cases that i'm thinking through and how this might be useful for us!)

just for reference, here's an example event payload from my local copy of servantes, where i have labels and disable resources explicitly enabled:
```json
  ...
  "tiltfile.feature.disable_resources": "true",
  "tiltfile.feature.labels": "true",
  "tiltfile.feature.live_update_v2": "true",
  "tiltfile.feature.snapshots": "true",
  ...
  "tiltfile.invoked.enable_feature": "2",
  "tiltfile.invoked.enable_feature.arg.msg": "2",
```

Reply:
i would be fine with adding feature tags to the 'up.running' event as in here:
https://github.com/tilt-dev/tilt/blob/master/internal/engine/analytics/analytics_reporter.go#L148
which is at as least consistent with existing patterns (i.e., the feature is a property of a running tilt instance rather than a property of the tiltfile load) and should be easier to query correctly